### PR TITLE
Fix Active Storage Overview guide [ci skip]

### PR DESCRIPTION
**Pull request by yhirano55**

### CODE REVIEW

**Comment by kamipo** [comment]
Can you also fix this line (Users -> User) in the commit?
https://github.com/rails/rails/pull/31484/files#diff-e0a7f9274451f039571959ee77f804baL221

File: guides/source/active_storage_overview.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ You can create a user with an avatar:
 ``` ruby
 class SignupController < ApplicationController
   def create
-    user = Users.create!(user_params)
+    user = User.create!(user_params)
```
Comment (yhirano55) [inline guides/source/active_storage_overview.md]: @kamipo Thanks for the comment. I've fixed this.

File: guides/source/active_storage_overview.md [hunk 2]
```diff
@@ -332,7 +332,7 @@ To enable variants, add `mini_magick` to your `Gemfile`:
 gem 'mini_magick'
 ```
 
-When the browser hits the variant URL, ActiveStorage will lazy transform the
+When the browser hits the variant URL, Active Storage will lazy transform the
```
Comment (kamipo) [inline guides/source/active_storage_overview.md]: Still have ActiveStorage in the guide.
https://github.com/rails/rails/pull/31484/files#diff-e0a7f9274451f039571959ee77f804baL511

Comment (yhirano55) [reply guides/source/active_storage_overview.md]: You have a point. Thanks. I've fixed it.

**Pull request merged**